admin/config-ui/class-configuration-page.php

```diff
@@ -104,8 +104,7 @@ public function show_wizard() {
 				printf(
 					/* translators: %s expands to Yoast SEO. */
 					__( '%s &rsaquo; Configuration Wizard', 'wordpress-seo' ),
-					'Yoast SEO'
-				);
+					'Yoast SEO' );
```

Comment:
There is a strange error regarding tabs, expected minimal `4` tabs, found `4`. Do you have any idea how that is possible?